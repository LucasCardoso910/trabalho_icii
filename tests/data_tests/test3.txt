8
19
12
08
45
43
56
67
95
test3
