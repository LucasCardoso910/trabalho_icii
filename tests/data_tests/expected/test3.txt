bubble_sort:28,12
heap_sort:35,58
insert_binary_sort:14,18
insert_sort:11,25
merge_sort:14,32
quick_sort:24,23
selection_sort:28,21
shake_sort:13,12
